src/pages/Pathway/Pathway.js

```diff
@@ -23,7 +23,10 @@ const Pathway = () => {
             {example_resources.map((resource, index) => 
                 <PathDay index={index + 1} resource={resource} />
             )}
-            
+            <h2>Come back later?</h2>
+            <p3>You can come back later and your pathway will be saved! Alternatively,
+                 you can <a href="/quiz" target="_blank" rel="noreferrer noopener">start over</a>.</p3>
```

Comment:
could we remove the `target="_blank"`? we don't actually want to open a new tab when we start over D:

Reply:
Hello, sorry about that, I must have been the reserved label. I was trying to contribute to an open source project for my class, but ill make sure to check out other ACM's open-source projects!
Anyways made the fix and deleted target="_blank".
Thanks!